package_test.go

```diff
@@ -581,6 +581,80 @@ func (s *PackageSuite) TestAllErrors(c *C) {
 	}
 }
 
+func (s *PackageSuite) TestTransactions(c *C) {
+	dropTables, sqldb, err := personAndAddressDB()
+	if err != nil {
+		c.Fatal(err)
```

Comment:
All of these `if err != nil .. Fatal` are better spelled as `c.Assert(err, IsNil)`.

Reply:
Changed.